data/json/items/comestibles/mutagens.json

```diff
@@ -0,0 +1,851 @@
+[
```

Comment:
I assume you renamed and committed "mutagen.json" file

Reply:
Whoops, I must have got confused when I added the serum and mutagen recipes. Thanks I'll fix it.